var/ramble/repos/builtin/applications/intel-mpi-benchmarks/application.py

```diff
@@ -38,14 +38,16 @@ class IntelMpiBenchmarks(SpackApplication):
     required_package('intel-mpi-benchmarks')
 
     executable('pingpong',
-               '-genv I_MPI_FABRICS=shm:tcp '
+               '-genv FI_PROVIDER=tcp '
```

Comment:
I think we don't want to specify `FI_PROVIDER`, and let modifiers set this for us.

The `tcp` provider is only available in some versions of MPI.

Reply:
I agree, I think we should probably remove all the `-genv` lines and let the user specify it at the ramble.yaml level 

Realistically there is a good chance these values are default on GCP, and don't make as much sense outside of GCP (or at least TCP networks)

Reply:
Done!